draft-ietf-ppm-dap.md

```diff
@@ -2449,62 +2389,23 @@ In the presence of this adversary, DAP aims to achieve the privacy and
 robustness security goals described in {{!VDAF}}'s Security Considerations
 section.
 
-Currently, the specification does not achieve these goals. In particular, there
-are several open issues that need to be addressed before these goals are met.
-Details for each issue are below.
-
-1. When crafted maliciously, collect requests may leak more information about
-   the measurements than the system intends. For example, the spec currently
-   allows sequences of collect requests to reveal an aggregate result for a
-   batch smaller than the minimum batch size. [OPEN ISSUE: See issue#195. This
-   also has implications for how we solve issue#183.]
```

Comment:
I deleted this because it's no longer true under the current collect protocol, which if anything might err too far on the side of protecting privacy at the cost of utility.